### ISSUE TITLE
Add logins support for scrubbing encrypted data

### ISSUE THREAD

**Issue by lougeniaC64** [mozilla/application-services#4066]
This will be very similar to what was done in #4029 for Logins with some of the work having already been implemented in #4064. This work should be based on the logins encryption branch and merged into it once this work has been approved.

Note: This ticket is blocked by #4065.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/SYNC-2633)
┆Epic: Logins update and sqlcipher migration
┆Fix Versions: Release 94
┆Sprint End Date: 2021-09-17

**Comment by data-sync-user** [comment]
➤ Ben Dean-Kawamura commented:

I'm going to push this one back to next sprint.  The main encryption issue is still in my local fork, I'd rather not start this until that one gets approved and merged.

**Comment by lougeniaC64** [comment]
We’re going to use the preexisting `wipe_local` function to remove local and mirror data, and the sync metadata instead of scrubbing the encrypted data from the records when the per-field encryption key is lost or corrupted. As such this bug can be closed.
